### ISSUE TITLE
[DOC]How to build a feature with multiple sources

### ISSUE THREAD

**Issue by jexterliangsufe** [feathr-ai/feathr#1143]
### Willingness to contribute

No. I cannot contribute to this at this time.

### Description of proposal (what needs changing)

I have read the docs and codes for building a feature. But your FeatureAnchor seams to have only a source. For example, I have a feature called "the number of Franch hotels ordered by the user" and two hive tables(one records the hotel ordered by the user, the other records the location of the hotel). How can I build the feature? use DerivedFeature? 
In other words, I want to know the basic requirements of data structure(batch source) for Feathr.
